README.md

```diff
@@ -1,2 +1,175 @@
-# go-libp2p-relay
-Circuit Switching for libp2p
+# go-libp2p-circuit-relay
+> Circuit Switching in libp2p
```

Comment:
The spec should be a libp2p spec inside the specs folder, it applies to any language (or at least, it should)